src/pages/timecard/Timecard.spec.jsx

```diff
@@ -80,6 +80,48 @@ describe('Shift spanning mutiple days', () => {
     });
   });
 
+  it('should display start and finish time on timecard when both times have been entered', async () => {
+    renderWithTimecardContext(<Timecard />, {
+      summaryErrors: {},
+      setSummaryErrors: jest.fn(),
+      timeEntries: [
+        {
+          timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
+          startTime: '2022-09-01 01:00:00+00:00',
+          finishTime: '2022-09-03 05:00:00+00:00',
+        },
+      ],
+      setTimeEntries: jest.fn(),
+      timecardDate: '',
+      setTimecardDate: jest.fn(),
+    });
+
+    expect(
+      screen.getByText((content) =>
+        content.includes('01:00 on 1 September to 05:00 on 3 September')
+      )
+    ).toBeTruthy();
+  });
+
+  it('should display Shift(continued) if timecard spans more than one date and we are not on start date', async () => {
```

Comment:
Can we add a similar specific test to show that continued doesn't show when it spans more that one day but we're on the start date.

Reply:
That's been added now

Reply:
Might be a bit better to also expect that "(continued)" is falsy in the new test.